ui/components/ui/nickname-popover/index.scss

```diff
@@ -0,0 +1,67 @@
+.nickname-popover {
+  &__popover-wrap {
+    height: 232px;
+    border-radius: 8px;
+    background: $ui-white;
+    display: flex;
+    justify-content: center;
+
+    .popover-header {
+      padding: 16px 16px 0 0;
+    }
+
+    .popover-content {
+      margin-top: -15px;
+      align-items: center;
+    }
+  }
+
+  &__address {
+    @include H4;
+
+    font-weight: bold;
+    display: flex;
+    align-items: center;
+    color: $Black-100;
+    padding-top: 8px;
+  }
+
+  &__public-address {
+    @include H7;
+
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    min-height: 25px;
+    background: $Grey-000;
+    border-radius: 40px;
+    padding-left: 8px;
+    padding-right: 2px;
+    margin-top: 8px;
+
+    button {
+      background: none;
+    }
+
+    &__constant {
+      @include H8;
+
+      color: $Grey-500;
+    }
+  }
+
+  &__view-on-block-explorer {
+    @include H7;
+
+    color: $Blue-500;
```

Comment:
This should be `$primary-1`, as it's in our new color system.

Reply:
@darkwing I change color to be in our new color system.
You can see changes here: 5e5bfb5241fa5092766be5a3381e99bdbacfb96d